src/OrientateToMouseIC.cpp

```diff
@@ -42,15 +42,8 @@ void OrientateToMouseIC::handleInput(const SDL_Event& _event) {
     if (yMouse > yWinSize)
         yMouse = yWinSize;
 
-    // Get father's position
-    TransformComponent* fatherTransform = dynamic_cast<TransformComponent*>(
-        father->getComponent("TransformComponent"));
-    Ogre::Vector3 fatherPos = fatherTransform->getPosition();
-    fatherPos.x += xWinSize / 2;
-    fatherPos.y += yWinSize / 2;
-
-    // Calculate angle between mouse and father
-    float angleInRad = atan2(fatherPos.y - yMouse, fatherPos.x - xMouse);
+    // Calculate angle between mouse and window center
+    float angleInRad = atan2(yWinSize / 2 - yMouse, xWinSize / 2 - xMouse);
```

Comment:
Just one remark (I know this is similar to the old code). Wouldn't it be more efficient/accurate to make both `2`s, floating-point? As in, `2.0f`.